YunoHost/yunohost
# Remove legacy settings

## The problem

Too many code to manage for old settings protected_uris, unprotected_uris, skipped_uris.

## Solution

Migrate all settings, to some permissions

## PR Status

Just tested qui unit test and it pass. It fail on the here probably because there still need some migrations.

## How to test

Checkout the branch. Try to install some app legacy app and see that some permission are made for the legacy settings.

## Validation
